client/components/PetTypesPage.js

```diff
@@ -0,0 +1,47 @@
+import React, { useState, useEffect } from 'react'
+import PetTypesItem from './PetTypesItem'
+
+const PetTypesPage = (props) => {
+    const [petTypes, setPetTypes] = useState([])
+    
+    const getPetTypes = async () => {
+        try {
+            const response = await fetch("/api/v1/pet-types")
+            if (!response.ok) {
+                const errorMessage = `${response.status} (${response.statusText})`
+                const error = new Error(errorMessage)
+                throw (error)
+            }
+            const responseBody = await response.json()
+            setPetTypes(responseBody.petTypes)
+        } catch (err) {
+            console.error(`Error in Fetch: ${err.message}`)
+        }
+    }
+
+    useEffect(() => {
+        getPetTypes()
+    }, [])
+
+    const petTypesData = petTypes.map((value) => {
+        return (
+            <PetTypesItem

```

Comment:
This might be easier to pass the entire petType as props to the PetTypesItem
```
petTypes.map(type => {
  return <PetTypesItem key={type.id} type={type} />
})
```
You can then destructure it in the PetTypesItem component.